--- trunk/libsoup/soup-auth-digest.c	2008/07/26 13:10:27	1143
+++ trunk/libsoup/soup-auth-digest.c	2008/07/26 13:18:57	1144
@@ -98,9 +98,7 @@
 SoupAuthDigestAlgorithm
 soup_auth_digest_parse_algorithm (const char *algorithm)
 {
-	if (!algorithm)
-		return SOUP_AUTH_DIGEST_ALGORITHM_NONE;
-	else if (!g_ascii_strcasecmp (algorithm, "MD5"))
+	if (!algorithm || !g_ascii_strcasecmp (algorithm, "MD5"))
 		return SOUP_AUTH_DIGEST_ALGORITHM_MD5;
 	else if (!g_ascii_strcasecmp (algorithm, "MD5-sess"))
 		return SOUP_AUTH_DIGEST_ALGORITHM_MD5_SESS;

