From: Tomas Popela <tpopela@redhat.com>
Date: Tue, 4 Sep 2018 11:38:22 +0200
Subject: meson : Add missing check for gmtime_r()

---
 meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/meson.build b/meson.build
index 6e5c4b3..9b0a9f1 100644
--- a/meson.build
+++ b/meson.build
@@ -94,6 +94,10 @@ endif
 libpsl_required_version = '>= 0.20'
 libpsl_dep = dependency('libpsl', version : libpsl_required_version)
 
+if cc.has_function('gmtime_r', prefix : '#include <time.h>', args : ['-D_POSIX_SOURCE'])
+    cdata.set('HAVE_GMTIME_R', '1')
+endif
+
 ###################
 # GIO TLS support #
 ###################
