From: Tomas Popela <tpopela@redhat.com>
Date: Tue, 4 Sep 2018 15:38:53 +0200
Subject: meson: Add missing libsoup-gnome headers when generating enums

---
 libsoup/meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libsoup/meson.build b/libsoup/meson.build
index b4dbb46..1b998a6 100644
--- a/libsoup/meson.build
+++ b/libsoup/meson.build
@@ -195,7 +195,10 @@ configure_file(input : 'soup-version.h.in',
 
 enum_types = 'soup-enum-types'
 soup_enums = gnome.mkenums('soup-enum-types',
-  sources : soup_installed_headers,
+  sources : [
+    soup_installed_headers,
+    soup_gnome_installed_headers
+  ],
   h_template : enum_types + '.h.template',
   c_template : enum_types + '.c.template',
   install_header : true,
