From: Tomas Popela <tpopela@redhat.com>
Date: Tue, 4 Sep 2018 15:34:40 +0200
Subject: meson: Remove an extra Gio include for the GNOME introspection

---
 libsoup/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libsoup/meson.build b/libsoup/meson.build
index 52cc543..9991bbc 100644
--- a/libsoup/meson.build
+++ b/libsoup/meson.build
@@ -327,7 +327,6 @@ if enable_introspection or enable_vapi
       export_packages : libsoup_gnome_api_name,
       extra_args : gir_gnome_args,
       dependencies : soup_gir_gen_dep,
-      includes : 'Gio-2.0',
       install : true,
       header: join_paths(meson.project_name(), 'soup-gnome.h'),
     )
